app/components/assets/bulk-assign-custody-dialog.tsx

```diff
@@ -16,20 +19,33 @@ export const BulkAssignCustodySchema = z.object({
 export default function BulkAssignCustodyDialog() {
   const zo = useZorm("BulkAssignCustody", BulkAssignCustodySchema);
 
+  const { isSelfService } = useUserRoleHelper();
+  const { rawTeamMembers } = useLoaderData<{ rawTeamMembers: TeamMember[] }>();
```

Comment:
Here you shouldnt hardcode the type like that. you should import the type of the loader from asset index. Also when I test this for me raw team members in empty. Maybe something is wrong in the process of inviting a self service user and the team member is not created properly. To recreate it:
1. Go to team
2. Invite new user with type self service. I use gmail with +self_service to manage this easily.

Can you please test this and lmk if it works. It needs to be further investigated.